File: Contracts/Model/Product/Response/ManagementDiscountsResponse.cs
﻿namespace Contracts.Model.Product.Response;
public class ManagementDiscountsResponse : ByBaseResponse
{
	public Guid Id { get; set; }
	public double Value { get; set; }
	public DateTimeOffset Start { get; set; }
	public DateTimeOffset End { get; set; }
	public ICollection<ProductsDiscountResponse>? ProductsDiscounts { get; set; }

}

File: ClientApp/src/app/core/interfaces/Product/Response/managementDiscountsResponse.ts
import { ByBaseResponse } from "../../byBaseResponse";
import { ProductsDiscountResponse } from "./productsDiscountResponse";

export interface ManagementDiscountsResponse extends ByBaseResponse {
    id: string;
    value: number;
    start: string;
    end: string;
    productsDiscounts: ProductsDiscountResponse[] | null;
}